.npmrc

```diff
@@ -1,7 +1,7 @@
 runtime = electron
 target_arch = x64
-brave_electron_version = 4.0.3
-chromedriver_version = 2.29
-target = v4.0.3
+brave_electron_version = 4.4.29
+chromedriver_version = 2.33
+target = v4.4.29
```

Comment:
thanks. muon is now on Chromium 62 so i think we should upgrade to that if possible. see https://github.com/brave/browser-laptop/commit/17018e721af2731a73257a01b10913166588c279 for instance